ood.yaml

```diff
@@ -1,4 +1,7 @@
 ---
 - hosts: ood
   roles:
+    - prep_ood
```

Comment:
Probably want to change this name to prep_ohpc_node or something similar. By breaking this into a generic role we are disassociating it from ood specific use